### PR TITLE
improve self signed certificate/ca details

### PR DESCRIPTION
**Pull request by bbodenmiller**
Improve instructions on how to add self signed certs/customer CAs. This is needed until https://gitlab.com/gitlab-org/omnibus-gitlab/issues/712 is addressed.

### CODE REVIEW

**Comment by bbodenmiller** [comment]
Also need to update https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/web_hooks/web_hooks.md.

File: doc/settings/configuration.md [hunk 1]
```diff
@@ -215,6 +215,38 @@ high_availability['mountpoint'] = '/var/opt/gitlab'
...
+  sudo nano /opt/gitlab/embedded/ssl/certs/certlink.sh
+  # Paste from https://gitlab.com/snippets/6285
+  sudo chmod +x /opt/gitlab/embedded/ssl/certs/certlink.sh
+  cd /opt/gitlab/embedded/ssl/certs/
+  sudo ./certlink.sh /etc/gitlab/CA/*
+  ```
+
+After the custom certificate is symlinked the errors should be gone and your custom certificate preserved on GitLab upgrades.
+
+As your certificates were placed in `/etc/gitlab/CA` they will be included in GitLab backups.
```
Comment (axilleas) [inline doc/settings/configuration.md]: The backup rake task does not backup `/etc/gitlab/`, it's up to the adminitrator to do this.

Comment (bbodenmiller) [inline doc/settings/configuration.md]: I'll fix this but that's silly. Backup should backup that folder.

Comment (ayufan) [inline doc/settings/configuration.md]: I think that it shouldn't, because /etc/gitlab contains encryption secrets, private keys and database secrets.

**Comment by bbodenmiller** [comment]
@axilleas @ayufan I've addressed the comments and squashed the commits.

File: doc/common_installation_problems/README.md [hunk 2]
```diff
@@ -265,29 +265,6 @@ PassThroughPattern: (packages\.gitlab\.com|packages-gitlab-com\.s3\.amazonaws\.c
 
 Read more about `apt-cacher-ng` and the reasons why this change is needed [on the packagecloud blog](http://blog.packagecloud.io/eng/2015/05/05/using-apt-cacher-ng-with-ssl-tls/).
 
-### Using self signed certificate or custom certificate authorities
```
Comment (maxlazio) [inline doc/common_installation_problems/README.md]: @bbodenmiller If we remove the header we break any existing links all around the internet. Can you leave the header and state where this section was moved?

Comment (maxlazio) [inline doc/common_installation_problems/README.md]: cc @axilleas

**Comment by maxlazio** [comment]
:+1: after addressing the line comment

File: doc/common_installation_problems/README.md (hunk 2 above)
Comment (axilleas) [inline doc/common_installation_problems/README.md]: :+1:

New users will go to new section, old links will not break.

**Pull request closed without merging**